### PR TITLE
Init logger before load operator config file

### PR DESCRIPTION
**Pull request by timdengyun**
Initializing logger before loading operator config file, otherwise, the log info will not be logged

Also, change log info level in validating config and go-fmt files

### CODE REVIEW

**Review by TaoZou1 — DISMISSED** [review]

File: cmd/main.go [hunk 1]
```diff
@@ -60,6 +60,7 @@ func init() {
 	if metrics.AreMetricsExposed(cf) {
 		metrics.InitializePrometheusMetrics()
 	}
+	log.Info("completed NSX Operator initialization")
```
Comment (dantingl) [inline cmd/main.go:63]: I do not think this log is needed

File: pkg/config/config.go [hunk 2]
```diff
@@ -235,11 +234,11 @@ func (nsxConfig *NsxConfig) validate() error {
 	}
 	tpCount := len(nsxConfig.Thumbprint)
 	if tpCount == 0 {
-		log.V(1).Info("no thumbprint provided")
+		log.Info("no thumbprint provided")
 		return nil
 	}
 	if tpCount == 1 {
-		log.V(1).Info("all endpoints share one thumbprint")
```
Comment (dantingl) [inline pkg/config/config.go:242]: I still think these two are V1 level..

File: cmd/main.go (hunk 1 above)
Comment (timdengyun) [reply cmd/main.go]: Agreed, removed.

File: pkg/config/config.go (hunk 2 above)
Comment (timdengyun) [reply pkg/config/config.go]: Revert to V(1)

**Review by TaoZou1 — APPROVED** [review]

**Pull request merged**